bundles/org.openhab.binding.openweathermap/src/main/java/org/openhab/binding/openweathermap/internal/handler/OpenWeatherMapOneCallHandler.java

```diff
@@ -512,7 +497,123 @@ private void updateHourlyForecastChannel(ChannelUID channelUID, int count) {
     }
 
     /**
-     * Update the channel from the last OpenWeatherMap data retrieved.
+     * Update the hourly forecast time series channel from the last OpenWeatherMap data retrieved.
+     * 
+     * @param channelUID the id identifying the channel to be updated
+     */
+    private void updateHourlyForecastTimeSeries(ChannelUID channelUID) {
+        String channelId = channelUID.getIdWithoutGroup();
+        String channelGroupId = channelUID.getGroupId();
+        if (forecastHours == 0) {
+            logger.warn(
+                    "Can't update channel group {} because forecastHours is set to '0'. Please adjust config accordingly",
+                    channelGroupId);
+            return;
+        }
+        if (channelId.equals(CHANNEL_TIME_STAMP) || channelId.equals(CHANNEL_CONDITION)
+                || channelId.equals(CHANNEL_CONDITION_ID) || channelId.equals(CHANNEL_CONDITION_ICON)
+                || channelId.equals(CHANNEL_CONDITION_ICON_ID)) {
```

Comment:
Note that I have decided to not create TimeSeries for non-Number channels, because obviously you cannot create a diagram for those.
However having those as TimeSeries data might still be useful for rules and scripts, I guess.

@jlaur WDYT?

Reply:
@jlaur Gentle ping, as IMHO this should make it into 4.1.
Would be great if there is more than one add-on (EnergiDataService) supporting the new time-series feature.

Reply:
I guess in order to fully replace the old work-around of having an enormous amount of channels to represent values at different points in time, you would need to include those non-number channels as well?

For reference: https://www.openhab.org/addons/bindings/openweathermap/#one-call-api-minutely-forecast

This is only about persistence, and as you mention, rules can also take advantage of this, not only charts. You could also access the data in a database from something completely outside openHAB.

Perhaps @J-N-K has an opinion on this?

Reply:
> I guess in order to fully replace the old work-around of having an enormous amount of channels to represent values at different points in time, you would need to include those non-number channels as well?

Agreed. I will also enable those channels (I hope to get to this tomorrow, now is a bit too late.)

Reply:
Implemented in https://github.com/openhab/openhab-addons/pull/15963/commits/32f404cb90f25659a24cf151abd53cb15b74a7bc.